templates/routing-legacy/main.go

```diff
@@ -74,15 +74,13 @@ func solver(
 		return FleetOutput{}, err
 	}
 
-	runSolutions := run.Last
```

Comment:
I am not completely following why we change the default output type of the routing legacy app. The issue doesn't have context either. Can you catch me up? :thinking:

Reply:
I think, I thought for a moment, that we returned all solutions by default in cloud. but that is incorrect. so I reverted this change.

Reply:
Oh yeah, makes sense. We did return all solutions for the frontend to visualize them. But in normal use, we did not. :relaxed: